File: Algorithm/Jungol/j1341/jungol_1341.c
// http://jungol.co.kr/bbs/board.php?bo_table=pbank&wr_id=2076&sca=20

#include <stdio.h>

int main(){
    int s, e, i;
    scanf("%d %d", &s, &e);

    if(s <= e){
        while(s <= e){
            for(i=1; i<=9; i++){
                printf("%d * %d = %2d   ", s, i, s*i);
                if(i%3 == 0) printf("\n");
            }
            printf("\n");
            s += 1;
        }
    }
    else if(s > e){
        while(s >= e){
            for(i=1; i<=9; i++){
                printf("%d * %d = %2d   ", s, i, s*i);
                if(i%3 == 0) printf("\n");
            }
            printf("\n");
            s -= 1;
        }
    }

    return 0;
}

File: Algorithm/Jungol/j1341/jungol_1341.py
# http://jungol.co.kr/bbs/board.php?bo_table=pbank&wr_id=2076&sca=20

s, e = input().split()
s = int(s)
e = int(e)

if(s <= e):
    while(s <= e):
        for i in range(1, 10):
            print(f"{s} * {i} = {s*i: >2}   ", end='')
            if(i % 3 == 0):
                print()
        print()
        s += 1
elif(s > e):
    while(s >= e):
        for i in range(1, 10):
            print(f"{s} * {i} = {s*i: >2}   ", end='')
            if(i % 3 == 0):
                print()
        print()
        s -= 1
